components/brave_wallet/browser/brave_wallet_service_factory.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/brave_wallet_service_factory.h"
+
+#include "brave/components/brave_wallet/browser/brave_wallet_service.h"
+#include "chrome/browser/profiles/incognito_helpers.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/keyed_service/content/browser_context_dependency_manager.h"
+
+// static
+BraveWalletServiceFactory* BraveWalletServiceFactory::GetInstance() {
+  return base::Singleton<BraveWalletServiceFactory>::get();
+}
+
+// static
+BraveWalletService* BraveWalletServiceFactory::GetForProfile(Profile* profile) {
```

Comment:
this factory needs to go in `brave/browser/brave_wallet`

Reply:
I moved that already in a diff PR a few weeks ago